src/Http/Controllers/Cp/ProductController.php

```diff
@@ -179,17 +179,15 @@ public function update(ProductUpdateRequest $request, Product $product)
     {
         $this->authorize('update', $product);
 
-        $validation = $request->validated();
```

Comment:
Why did you remove the request validation?

Reply:
There's no need to call that method. All it does is return the validated data, not actually validate the request, that gets done during the request lifecycle.

Reply:
Ah ok, I always thought that was required 😃